GatherPress/gatherpress
# Use date & time format defaults from options

As I installed GatherPress the first time I was a little surprised about the date and time formats used within blocks and the editor.

I expected them to align with what I already defined for the `date_format` and `time_format` core options.

### Description of the Change


Inherit the default values for gatherpress' date & time formats from the related blog options.


~~Closes #~~

### How to test the Change


1. Got to **General > Settings**
2. Change your date & time formats to anything you like
3. Go to **Event > Settings**
4. Delete every custom date and time format you (may have) defined in the past
5. Reload the settings page to make sure the new defaults align with your blog options.

### Changelog Entry

> Changed - Get defaults for date & time format from default (blog) options.

### Credits

Props @carstingaxion 

### Checklist:
